extras/data/COMPASS/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+COMPASS:
+  description: Measurements with Polarised beams
```

Comment:
maybe we can find a more concise description ... and don't copy-and-paste everywhere ...

Reply:
I'm not really sure about that... 
Maybe we should drop this metadata file as this info are now in the new commondata metadata ?

Reply:
well, this is the theory side and there is no 1:1 mapping for data and theory - especially for $A_1$ - so, I think we still need that; you can get some inspiration from there ...

Reply:
I disagree, there is a quite 1:1 mapping...

Reply:
not for $A_1$! for $A_1$ you need **two** grids/FK tables and then you make a ratio between those two; one of them will say it is polarized and the other not ...

Reply:
Sure but here we have a single metadata and yaml file, because the kinematic is the same.

Reply:
yes, kinematics are the same - but still we are on the theory side here (despite the name of the folder being `data`), so for `yadism` this is just a coincidence! it would be just wrong to have a grid with metadata "description: blub polarized beams" and "y_label: g1" and then you have to fold it with a unpolarized proton PDF and you get F1 ... this folder here is producing pinecards and they will be transferred to the official pinecards repo and there we have to be compliant with pineappl (because yadism at this point is hidden behind pinefarm)